form/bootstrap4.rst

```diff
@@ -0,0 +1,88 @@
+Bootstrap 4 Form Theme
+======================
+
+Symfony provides several ways of integrating Bootstrap into your application. The
+most straightforward way is to just add the required ``<link>`` and ``<script>``
+elements in your templates (usually you only include them in the main layout
+template which other templates extend from):
+
+.. code-block:: html+twig
+
+    {# templates/base.html.twig #}
+
+    {# beware that the blocks in your template may be named different #}
+    {% block head_css %}
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
I am not sure if we really should include this code example here. It will become outdated really fast and people will still continue to copy and paste it. What about linking to the [Bootstrap getting started docs](https://getbootstrap.com/docs/4.0/getting-started/introduction/) instead?